Server/rainbow-server/Sources/Application/Routes/ScoreRoutes.swift

```diff
@@ -54,10 +53,18 @@ func addNewEntry(newEntry: ScoreEntry, completion: @escaping(ScoreEntry?, Reques
     }
 }
 
-func updateEntry(anonymousIdentifier: String, newEntry: ScoreEntry, completion: @escaping (ScoreEntry?, RequestError?) -> Void) {
-    // TODO
-}
-
-func deleteEntry(anonymousIdentifier: String, completion: @escaping(RequestError?) -> Void) {
-    // TODO
+func updateEntry(anonymousIdentifier: String,newEntry: ScoreEntry, completion: @escaping (ScoreEntry?, RequestError?) -> Void) {
+    Log.debug("Updating entry document")
```

Comment:
We should log this statement as `Log.info()`. Let's change only this then I'll merge and add plenty more logging where we should 😄

Reply:
sounds good.